src/app/login/login.component.html

```diff
@@ -5,90 +5,62 @@
 
-  <div class="ui white header">
-    <p class="nomargin">Tell the Whole Tale about your research through powerful computing tools</p>
-    <p class="nomargin">like Jupyter and RStudio, or craft your own.</p>
+    <div class="ui white header">
+        <p class="nomargin">Empower and share your research with Whole Tale, an open source platform for</p>
+        <p class="nomargin">reproducible computational research. <a href="https://wholetale.org" style="color:white;"><u>Learn more...</u></a></p>
+    </div>
   </div>
 
   <a class="ui large blue button" (click)="loginWithGlobus()">
     <div class="access-wholetale">Access</div>
     <img src="../../assets/img/whole_tale_logo_text_long_light_large.png" width="200px" style="width:200px">
   </a>
 
-  <div class="login icon backdrop">
-    <div class="ui grid">
-      <div class="center aligned four wide column">
-        <i class="huge disk icon"></i>
-        <div class="ui large header">
-          <p class="botmargin">Registered</p>
-          <p class="nomargin">Data</p>
-        </div>
-      </div>
-
-      <div class="two wide column">
-      </div>
-
-      <div class="center aligned four wide column">
-        <i class="huge dashboard icon"></i>
-        <div class="ui large header">
-          <p class="botmargin">Analyze/</p>
-          <p class="nomargin">Compute</p>
-        </div>
-      </div>
-
-      <div class="two wide column">
-      </div>
+  <p style="botmargin" style="color:black;">By accessing Whole Tale, you agree to our
+    <a href="https://docs.wholetale.org/tos" style="color:black"><u>Terms of Use.</u></a>
+  </p>
 
-      <div class="center aligned four wide column">
-        <i class="criss cross icon"></i>
-        <div class="ui large header">
-          <p class="botmargin">Published</p>
-          <p class="nomargin">Results</p>
-        </div>
-      </div>
-    </div>
-  </div>
-
-  <div class="accordion-container">
-    <div class="ui accordion">
-      <div class="title">
-        <i class="dropdown icon"></i>
-        <a class="ui white header">Login questions?</a>
-      </div>
-      <div class="content">
-        <div class="ui raised segment">
-          <div class="ui header header">
-              In order to use the Whole Tale, you need to be able to sign in using Globus. Click below to find out more.
-          </div>
-          <a class="ui button" target="_blank" href="https://wholetale.readthedocs.io/en/stable/users_guide/sign_in.html">Signing In</a>
-        </div>
-      </div>
-    </div>
+  <div style="color:#666">
+    <p class="nomargin" style="margin-top: 60px; font-size: 14pt;">Whole Tale is supported by the National Science Foundation under Grant No. OAC-1541450.</p>
+    <p class="nomargin" style="font-size: 14pt;">Computational resources are provided by XSEDE through Jetstream.</p>
+    <p class="botmargin" style="font-size: 14pt;">Access to Stata is provided by StataCorp LLC.</p>
   </div>
```

Comment:
I think black looks better here and a little smaller:

```suggestion
  <div style="color:black">
    <p class="nomargin" style="margin-top: 50px; font-size: 12pt;">Whole Tale is supported by the National Science Foundation under Grant No. OAC-1541450.</p>
    <p class="nomargin" style="font-size: 12pt;">Computational resources are provided by XSEDE through Jetstream.</p>
    <p class="botmargin" style="font-size: 12pt;">Access to Stata is provided by StataCorp LLC.</p>
  </div>
```